aws_cloudfront_distribution.tf

```diff
@@ -14,7 +14,7 @@ resource "aws_cloudfront_distribution" "s3-distribution-TLS-v1" {
     }
   }
 
-  enabled             = true
+  enabled = true
 
   default_cache_behavior {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Cloud Front distribution does not have web application firewall enabled**
 
**Vulnerability**
As cloud-front is a public facing service. Thus, having no web application firewall makes it vulnerable to application layer attacks
 
**Remediation**
Ensure that the Web application firewall is enabled at cloud front distribution

source: accurics-